.github/workflows/ci.yaml

```diff
@@ -20,7 +20,7 @@ permissions:
 # additional changes
 concurrency:
   group: ${{ github.workflow }}-${{ github.ref }}
-  cancel-in-progress: ${{ github.event_name == 'pull_request' }}
+  cancel-in-progress: ${{ github.event_name == 'push' }}
```

Comment:
Will this cancel pushes on the `main` branch too? I would expect us to want all pushes to be fully tested there.

Reply:
Yes you are right. It will also cancel on main. May be we should revert #9017